src/java_bytecode/java_bytecode_language.cpp

```diff
@@ -385,26 +407,9 @@ void java_bytecode_languaget::replace_string_methods(
 
 bool java_bytecode_languaget::final(symbol_tablet &symbol_table)
 {
-  java_internal_additions(symbol_table);
-
   // replace code of String methods calls by code we generate
   replace_string_methods(symbol_table);
-
-  main_function_resultt res=
-    get_main_symbol(symbol_table, main_class, get_message_handler());
-  if(res.stop_convert)
-    return res.error_found;
-
-  // generate the test harness in __CPROVER__start and a call the entry point
-  return
-    java_entry_point(
-      symbol_table,
-      main_class,
-      get_message_handler(),
-      assume_inputs_non_null,
-      max_nondet_array_length,
-      max_nondet_tree_depth,
-      get_pointer_type_selector());
+  return false;
```

Comment:
Would you mind adding a comment to this?
Sometime later it might not be obvious why this cannot fail.